src/web/jcc/client/components/TaskSettingsPanel/__tests__/TaskSettingsPanel.test.tsx

```diff
@@ -0,0 +1,64 @@
+import { render, screen, waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+import { TaskSettingsPanel, Props } from "../TaskSettingsPanel";
+import { MissionTask, TaskType } from "../../shared/JAIAProtobuf";
+import { validateTask } from "../../../../../tests/helpers/ValidateTask";
+
+let mockProps: Props = {
+    isEditMode: true,
+    enableEcho: false,
+    onChange: (task?: MissionTask) => mockOnChange(task),
+};
+
+//Mock of the onChange callback to update Props
```

Comment:
Nitpicking: Should we capitalize and add a space with the comment? Not sure just want it to be consistent.